### PR TITLE
Change path to point to proper endpoint

### PR DESCRIPTION
**Pull request by sambible**
Small update to Ansible Nailgun PR recently merged, this properly sets the path to the endpoint it needs to point to.

### CODE REVIEW

File: tests/test_entities.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def test_id_and_which(self):
             (entities.Host, 'packages'),
             (entities.Host, 'puppetclass_ids'),
             (entities.Host, 'smart_class_parameters'),
-            (entities.Host, 'list_ansible_roles'),
+            (entities.Host, 'ansible_roles'),
```
Comment (adarshdubey-star) [inline tests/test_entities.py:328]: @sambible can you please try running this test case locally and see if that works for you, I purposely changed that to `list_ansible_roles` as I think we are putting the methods here and it worked for me.

Comment (sambible) [reply tests/test_entities.py:328]: Yea, I've run this locally - in the path function it needs to be the actual name of the endpoint in Satellite - which is a GET on ansible_roles, not list_ansible_roles

**Review by adarshdubey-star — APPROVED** [review]

File: tests/test_entities.py (hunk 1 above)
Comment (adarshdubey-star) [reply tests/test_entities.py:328]: PRT run passed with these changes in my open PR. Good catch @sambible

**Review by jyejare — APPROVED** [review]

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, please add specific branch labels where you want this to be cherrypicked, also open manual cpick for #841

**Pull request merged**